src/main/java/org/apache/commons/csv/CSVParser.java

```diff
@@ -147,7 +147,7 @@ private CSVRecord getNextRecord() {
             try {
                 return CSVParser.this.nextRecord();
             } catch (final IOException e) {
-                throw new UncheckedIOException(e.getClass().getSimpleName() + " reading next record: " + e.toString(), e);
+                throw new UncheckedIOException("Error in reading next record: " + e.toString(), e);
```

Comment:
Why not just `throw new UncheckedIOException("Error in reading next record", e);`?  No duplication. Plain and simple.

Reply:
Might be the intension was to provide the sense of error in the message itself.